vispy/app/backends/_qt.py

```diff
@@ -143,7 +149,7 @@ def message_handler(*args):
 
     if qt_lib in ("pyqt4", "pyside"):
         msg_type, msg = args
-    elif qt_lib == "pyqt5":
+    elif qt_lib == ("pyqt5", "pyside2"):  # Is this correct for pyside2?
```

Comment:
Don't know more details, but `==` definetly doesn't work here.

`elif qt_lib in ("pyqt5", "pyside2"):`

Reply:
Wow, good catch. @hmaarrfk would you mind changing that?